Bottom: fd70d97eaa5d7e96d7a21f84e041b300e0b79589
Top:    4822c077974da15d8c7a00213635a4e29e122796
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-25 15:24:11 -0500

Create certificate directories during image build


---
diff --git a/docker/docker-svn/Dockerfile b/docker/docker-svn/Dockerfile
index 967bca5..7397d68 100644
--- a/docker/docker-svn/Dockerfile
+++ b/docker/docker-svn/Dockerfile
@@ -10,6 +10,7 @@ MAINTAINER Joseph Lutz <Joseph.Lutz@novatechweb.com>
 RUN apt-get update && \
   DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
     apache2 \
+    ca-certificates \
     enscript \
     libapache2-svn \
     php5-common \
diff --git a/docker/docker-svn/configure.sh b/docker/docker-svn/configure.sh
index fab90ad..bff6502 100755
--- a/docker/docker-svn/configure.sh
+++ b/docker/docker-svn/configure.sh
@@ -6,6 +6,10 @@ SVN_BASE_DIR=/var/lib/svn
 # Create a directory for a dead symlink
 mkdir /var/cache/websvn/tmp
 
+# Create certificate directories
+mkdir -p /etc/grid-security/certificates
+mkdir -p /etc/ssl/private
+
 # comment out apache2 config file lines that reference the environment variables
 sed -i 's|^Mutex file|#Mutex file|' /etc/apache2/apache2.conf
 sed -i 's|^PidFile |#PidFile |' /etc/apache2/apache2.conf
